packages/styles/components/_c.toggle.scss

```diff
@@ -93,6 +95,7 @@
           "control-cross-16",
           $color-toggle-disabled-background
         ));
+        background-size: contain;
```

Comment:
~~background-size: contain;~~ => Due to the "cascade", it is not necessary to repeat this value here.